ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -214,6 +236,11 @@ export default defineComponent({
     tbody > tr.v-row-group__header:hover {
       background: #e2e8ee !important;
     }
+
+    .no-owners-error {
+      text-align: center;
+      color: $error;
+    }
   }
```

Comment:
I'm not sure you need a unique error class here, `.error-text` in the base.scss handles the color and there is a vuetify helper class `text-center` to help with alignment.